src/platforms/ios/iosgleanbridge.swift

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Glean
+
+public class IOSGleanBridgeImpl : NSObject {
+  @objc init(telemetryEnabled isTelemetryEnabled: Bool, channel appChannel: String) {
+      Glean.shared.registerPings(GleanMetrics.Pings.self)
+      Glean.shared.initialize(
+        uploadEnabled: isTelemetryEnabled,
+        configuration: Configuration.init(channel: appChannel),
+        buildInfo: GleanMetrics.GleanBuild.info
+      )
```

Comment:
By initializing Glean on iOS we are using all the goodnes that comes with the Glean iOS SDK, such as a ping upload worker that is not immediatelly killed when the application goes to background.

Reply:
This also means a new application id on iOS, just like in Android. I filed a ticket to register that https://mozilla-hub.atlassian.net/browse/VPN-3237